frontend/src/k8sTypes.ts

```diff
@@ -55,13 +55,19 @@ export type StorageClassConfig = {
   description?: string;
 };
 
+export enum Annotation {
```

Comment:
if you are exporting this, i would be more specific about what annotations these are. Something like CommonAnnotations might be fine. but could be anything

Reply:
I went with `MetadataAnnotation` to be more specific. Outside of that, they are k8s annotations used for metadata and they're in a file called k8sTypes, so I think that should suffice.